module/zfs/vdev_label.c

```diff
@@ -1088,8 +1089,13 @@ vdev_uberblock_load_impl(zio_t *zio, vdev_t *vd, int flags,
 		vdev_uberblock_load_impl(zio, vd->vdev_child[c], flags, cbp);
 
 	if (vd->vdev_ops->vdev_op_leaf && vdev_readable(vd)) {
+		/*
+		 * If the pool was last imported by a non-MMP aware ZFS,
+		 * the "MMP BLOCKS" may contain valid uberblocks.
+		 */
+		int slots = VDEV_UBERBLOCK_COUNT(vd) + MMP_BLOCKS_PER_LABEL;
```

Comment:
I've noticed that in a few spots we are simply adding the MMP  block count back. Couldn't we just introduce a different macro that would be equally clear? Something like `VDEV_UBERBLOCK_SLOTS` maybe?